lib/components/playground/dynamic-live.tsx

```diff
@@ -28,6 +28,7 @@ const DynamicLive: React.FC<Props> = ({ code, scope }) => {
           display: flex;
           flex-direction: column;
           box-sizing: border-box;
+          overflow-x: scroll;
```

Comment:
In scenarios where `x` is set to the scroll state but `y` is `unset`, `y` may also allow scrolling.
We need to add the judgment to apply this styles only at `xs` breakpoints.

Reply:
How about using `auto`? Would that work?